eth/eventdatahandler/event_data_handler.go

```diff
@@ -105,30 +106,30 @@ func (edh *EventDataHandler) HandleBlockEventsStream(blockEventsCh <-chan eventb
 		logger = logger.With(fields.Count(len(tasks)))
 		logger.Info("executing tasks")
 
-		// TODO:
-		// 1) find and remove opposite tasks (start-stop, stop-start, liquidate-reactivate, reactivate-liquidate)
-		// 2) find superseding tasks and remove superseded ones (updateFee-updateFee)
-		for _, task := range tasks {
-			edh.logger.Debug("going to execute task") // TODO: add more task details
-			if err := task(); err != nil {
+		cleanTaskList := cleanTaskList(tasks)
+		for _, task := range cleanTaskList {
+			edh.logger.With(
+				zap.String("event_type", task.GetEventType()),
+			)
+			edh.logger.Debug("going to execute remote task")
```

Comment:
```suggestion
			logger.Debug("going to execute remote task")
```

Reply:
Why do you call it "remote"?

Reply:
Now I see that you meant tasks for non-local events. I think we don't need to differentiate between those tasks. Let's call them all just tasks